test/oauth_proxy_outbound_test.js

```diff
@@ -21,7 +21,7 @@ describe('oauth_proxy outbound message integrity: verbs', function() {
     ['GET', 'DELETE'].forEach(function(verb) {
```

Comment:
All the failures are here:
```
8) should accept a properly signed DELETE over IPv6 with query via https
Error: Protocol "https:" not supported. Expected "http:".
    at new ClientRequest (_http_client.js:53:11)
    at Object.exports.request (http.js:31:10)
    at Object.exports.request (https.js:163:15)
    at Array.stream (/Users/wrb/tmp/oauth_reverse_proxy/node_modules/http-proxy/lib/http-proxy/passes/web-incoming.js:108:74)
    at ProxyServer.<anonymous> (/Users/wrb/tmp/oauth_reverse_proxy/node_modules/http-proxy/lib/http-proxy/index.js:80:21)
```

I'm not sure what might have changed between this version and master.  Looks like an https request is being sent with http.js.  Not sure why that would happen.